modules/release-notes/pages/relnotes.adoc

```diff
@@ -546,6 +546,8 @@ WARNING: Changing the internal replication setting will cause all XDCR processes
 | https://issues.couchbase.com/browse/MB-32685[MB-32685^]
 | *Summary*: Introduced a delay in updating the RBAC database revision number to avoid returning AUTH_STALE during a refresh of the RBAC database.
 
+| https://issues.couchbase.com/browse/MB-32497[MB-32497^]
+| *Summary*: Connection::signalIfIdle may cause a deadlock when used from a different thread
```

Comment:
I've not worked on server release notes, but this looks more like the initial problem statement?
Would it be better to reword as what has *changed* (e.g. the improvement made)

(Please ignore if I've missed the point/convention for server!)